src/components/app.scss

```diff
@@ -1,7 +1,9 @@
 @import "global-variables";
 
 // FIXME: Make this automatically distro-specific
-@import "../branding/fedora";
+/* @import "../branding/fedora"; */
+/* @import "../branding/centos"; */
+@import "../branding/bazzite";
```

Comment:
Here's the part where it should be automatic. Manually importing is fine for testing, of course. We want to make sure it's not something other than Fedora (either automatic or even manual for now) when merging, however.